### DISCUSSION TITLE
MayaUsd or UsdMaya?

### DISCUSSION THREAD

**Discussion by rlei-weta** [Autodesk/maya-usd#879] [Q&A] [answered]
Hello - in the code, there are classes and functions prefixed with a mixture of `MayaUsd` and `UsdMaya`.  

I'm getting the feeling that `MayaUsd` is the canonical prefix and `UsdMaya` might be a historical thing - but was hoping to hear confirmation from the maya-usd team :-)

This will help guide us naming our internal projects, plugins, and classes.  

Cheers
Richard

**Comment by ppt-adsk** [comment, marked as answer]
Hey @rlei-weta, thanks for writing.  UsdMaya is the prefix for older code brought over from the Pixar code base, and MayaUsd is the new Autodesk prefix.  Cheers.

**Reply by rlei-weta** [reply]
Thanks for the confirmation @ppt-adsk!
